javascript/javascript2/week1/homework/hyf-bay/main.js

```diff
@@ -2,3 +2,19 @@ console.log("Script loaded");
 
 const products = getAvailableProducts();
 console.log(products);
+
+function renderProducts(products) {
+  const ulElement = document.getElementById("products");
+  for (const product of products) {
+    ulElement.appendChild(createLiElementWithText(product.name));
+    ulElement.appendChild(createLiElementWithText(`Price : ${product.price}`));
+    ulElement.appendChild(createLiElementWithText(`Rating: ${product.rating}`));
+  }
+}
```

Comment:
How can you ensure, that the product name is of h1 format, like the example in codesandbox? 😊

> https://codesandbox.io/s/hyf-bay-first-week-oml13

Perhaps you can work a bit with the HTML tags and ad some list items with different styling? 💡

> https://www.w3schools.com/tags/tag_ul.asp

Reply:
I have done that in the css, so that every third li will be bolder. of course can be done from javascript

Reply:
Cool, it didn't show when I compiled your code - it could definitely be a miss on my part. Good to know you implemented this 😊